components/Nav/Social.js

```diff
@@ -5,7 +5,7 @@ import rem from '../../utils/rem'
 import { navbarHeight } from '../../utils/sizes'
 import Link from '../Link'
 
-const Wrapper = styled.nav`
+const Wrapper = styled.div`
```

Comment:
Was this on purpose?

Reply:
Yea, we already have a `nav` for all of the navbar AFAIK. There's no need to nest multiple `nav`s!